packages/wdio-utils/package.json

```diff
@@ -27,5 +27,6 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "types": "utils.d.ts"
```

Comment:
Can't we just point to `./build/index.d.ts`?

Reply:
Unfortunately not no, we then get the following error during `npm run setup-full` on `npx tsc -b ...`:
```
error TS2688: Cannot find type definition file for '@wdio/utils'.
```

Reply:
we just moved this package over to typescript and it has types defined now

Reply:
Ah great, let me rebase my branch then.

Reply:
still getting the error:
```
error TS2688: Cannot find type definition file for '@wdio/utils'.
```
(see builds here)